pkg/clients/edgeconnect/endpoints.go

```diff
@@ -2,10 +2,30 @@ package edgeconnect
 
 import "fmt"
 
+// EdgeConnect API
+
+func (c *client) getEdgeConnectApiUrl() string {
+	return c.baseURL + "/platform/app-engine/edge-connect/v1"
+}
+
 func (c *client) getEdgeConnectsUrl() string {
-	return c.baseURL + "/edge-connects"
+	return c.getEdgeConnectApiUrl() + "/edge-connects"
 }
 
 func (c *client) getEdgeConnectUrl(id string) string {
-	return fmt.Sprintf("%s/edge-connects/%s", c.baseURL, id)
+	return fmt.Sprintf("%s/%s", c.getEdgeConnectsUrl(), id)
+}
+
+// Environment API
+
+func (c *client) getEnvironmentApiUrl() string {
+	return c.baseURL + "/platform/classic/environment-api/v2"
+}
+
+func (c *client) getSettingsObjectsUrl() string {
+	return c.getEnvironmentApiUrl() + "/settings/objects"
+}
+
+func (c *client) getSettingsObjectsIdUrl(objectId string) string {
+	return fmt.Sprintf("%s/%s", c.getSettingsObjectsUrl(), objectId)
```

Comment:
Neat: can we use the same style of string concatenation? 2 functions using `+`, and the last one is Sprintf.